xskillscore/core/comparative.py

```diff
@@ -167,3 +177,141 @@ def _categorical_metric(observations, forecasts, dim):
     walk.coords["alpha"] = alpha
     walk = _add_as_coord(walk, confidence, "confidence")
     return walk
+
+
+def mae_test(
+    forecasts1,
+    forecasts2,
+    observations=None,
+    dim=[],
+    time_dim="time",
+    alpha=0.05,
+):
+    """
+    Returns the Jolliffe and Ebert MAE significance test whether forecasts1 and
+    forecasts2 have different mean absolute error (MAE) at significance level alpha.
+    https://www.cawcr.gov.au/projects/verification/CIdiff/FAQ-CIdiff.html
```

Comment:
You'll want a one-line statement at the beginning to get this to render right in the docs. See what happens, e.g. in `_m2e`: https://climpred.readthedocs.io/en/stable/comparisons.html.

Also make sure to update the API with this.